lib/Wikia/src/Service/User/Attributes/UserAttributes.php

```diff
@@ -13,20 +13,15 @@ class UserAttributes {
 	/** @var string[string][string] */
 	private $attributes;
 
-	/** @var string[string] */
-	private $defaultAttributes;
-
 	/**
 	 * @Inject({
 	 *    Wikia\Service\User\Attributes\AttributeService::class,
-	 *    Wikia\Service\User\Attributes\UserAttributes::DEFAULT_ATTRIBUTES
 	 * })
 	 * @param AttributeService $attributeService
 	 * @param string[string] $defaultAttributes
 	 */
-	public function __construct( AttributeService $attributeService, $defaultAttributes ) {
+	public function __construct( AttributeService $attributeService ) {
```

Comment:
This class used to handle additional logic when it came to default options. That logic is now all handled by the User class so we can remove default options from here.